### PR TITLE
Platinum now spawns in maint

### PR DESCRIPTION
**Pull request by Jamini**
## About The Pull Request

Adds platinum to maint loot drop pools and broken autolathes

## Why It's Good For The Game

Adds a source of rare materials for high-end batteries to maint.

## Changelog
```changelog
balance: Added some platinum to maint spawns
```

### CODE REVIEW

**Review by zackerycodestoo — DISMISSED** [review]
There are times when I've been craving borosilicate glass. Thank you

File: code/modules/materials/material_sheets.dm [hunk 1]
```diff
@@ -197,8 +197,8 @@
 /obj/item/stack/material/platinum/random
 	rand_min = 1
 	rand_max = 10
-	//spawn_tags = SPAWN_TAG_MATERIAL_RESOURCES_RARE
-	//rarity_value = 45
+	spawn_tags = SPAWN_TAG_MATERIAL_RESOURCES_RARE  //Occulus edit: Re-adding platinum glass to spawn pools
```
Comment (Michiyamenotehifunana) [inline code/modules/materials/material_sheets.dm:200]: Move this to the non-rare tag so science doesn't get cursed with a useless material at roundstart.

**Review by Michiyamenotehifunana — CHANGES_REQUESTED** [review]
REEEEEE science shouldn't spawn with phoron glass.

**Review by Michiyamenotehifunana — APPROVED** [review]
Looks good.

**Pull request merged**